airbyte-integrations/connectors/source-stripe/source_stripe/schemas/payment_intents.json

```diff
@@ -0,0 +1,191 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "object": {
+      "type": ["null", "string"]
+    },
+    "amount": {
+      "type": ["null", "integer"]
+    },
+    "amount_capturable": {
+      "type": ["null", "integer"]
+    },
+    "amount_received": {
+      "type": ["null", "integer"]
+    },
+    "application": {
+      "type": ["null", "string", "object"],
+      "properties": {}
```

Comment:
Why all `properties` objects are blank there?

Reply:
Some of them don't include descriptions in the documentation.